src/preamble.js

```diff
@@ -53,7 +53,6 @@ if (typeof WebAssembly != 'object') {
 // Wasm globals
 
 var wasmMemory;
-var wasmExports;
```

Comment:
Where is this defined if not here?

Reply:
emscripten.py declares this, on line 818

Reply:
Ah, I missed that somehow, thanks.